apis/python/src/tiledbsoma/__init__.py

```diff
@@ -6,7 +6,7 @@
 Python package is an implementation of SOMA using the
 [TileDB Embedded](https://github.com/TileDB-Inc/TileDB) engine.
 
-Provides:
+Provides
```

Comment:
For on-line Python CLI help, I think with or without trailing `:` is OK

But for readthedocs output (see also [here](https://github.com/single-cell-data/TileDB-SOMA/wiki/Doc-work-and-sandbox-previews)) these show up as subsection titles wherein the `:` looks odd

Hence my removing the trailing `:`

Reply:
Google style guide implies (by example) that the colon is to be included in section headers, and does not use underline.

[Section 3.8](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings)

I have no personal opinion other than consistency is goodness

Reply:
SInce has been rebased atop Google-style PRs